Makefile

```
@@ -83,9 +79,19 @@ test-docs: ## Run documentation tests
 
 # --- Integration testing -------------------------------------------------------------------------
 
+.PHONY: testing-node-start
+testing-node-start: ## Start the test server
+	RUST_LOG=none cargo run --release --package node-builder > /dev/null &
```

This will fail silently if for some reason the build fails, right? Would be nice to be able to detect those cases so that the CI stops immediately.